ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1214,18 +1217,21 @@ def test_ipa_filecheck_bad_owner(self, modify_permissions):
         assert returncode == 1
         for check in data:
             assert check["result"] == "WARNING"
-            assert check["kw"]["key"] == '_etc_resolv.conf_owner'
-            assert check["kw"]["type"] == 'owner'
-            assert check["kw"]["expected"] == 'root'
-            assert check["kw"]["got"] == 'admin'
+            assert check["kw"]["key"] == "_etc_resolv.conf_owner"
+            assert check["kw"]["type"] == "owner"
+            assert check["kw"]["expected"] == "root"
+            assert check["kw"]["got"] == "admin"
```

Comment:
single vs double quotes. Not sure worth changing.

Reply:
Have changed to single quotes for all such tests.